doc/api/events.markdown

```diff
@@ -137,10 +137,17 @@ Execute each of the listeners in order with the supplied arguments.
 Returns `true` if event had listeners, `false` otherwise.
 
 
+### emitter.listenerCount(type)
+
+* `type` {String} The type of event
```

Comment:
The check should've been removed: https://github.com/nodejs/node/pull/2349#discussion_r37452127